src/styles/main.scss

```diff
@@ -0,0 +1,19 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Move `margin` and `padding` outside of `*` selector. You can use in `*` just `box-sizing` rule. 
Use it for all tags you need:
```
body,
h1,
p,
etc... {
  margin: 0;
  padding: 0;
}
```
But here you need just for `body` I think.